### ISSUE TITLE
Add system timestamp before each line in build logs

### ISSUE THREAD

**Issue by lamphanviet** [travis-ci/travis-ci#4968]
I can't find any options or setting to display the timestamp for each lines of the build logs.
For each line in the build logs, it would be great travis automatically insert the timestamp, looks like this:
```
[2015-10-16 08:32:20] line 1
[2015-10-16 08:32:23] line 2 ...
[2015-10-16 08:32:25] line n ...
```
With this timestamp information, we can know how long it takes to run that part of code, the time when the build hang,...

**Comment by joshk** [comment]
Thanks for the suggestion. I do like this idea. I'm not sure where it fits in our backlog though. I'll keep this open for the time being so we can discuss it within the team.

**Comment by lamphanviet** [comment]
Yes, I really appreciate that. There're a few other CIs already supporting this automatically.
I would love to see this feature coming out soon in the future.

**Comment by kuahyeow** [comment]
May I add my keen-ness for this as well - for reference see this Jenkins plugin https://plugins.jenkins.io/timestamper - you can switch between elapsed time, and wall clock time. So useful for debugging performance issues

**Comment by simonhf** [comment]
I would also like to add my support for this feature!

**Comment by ssbarnea** [comment]
Probably Travis teams excels at guesstimating the lines that take too much time to run, so they don't need to know the timestamps.
